R/errorChecking.R

```diff
@@ -1,19 +1,10 @@
+#' Check response of API call, produce warning on error
+#' 
+#' @param response httr response object from of API call
+
 checkHttpErrors <- function(response) {
 
   if(response["status_code"] != 200) {
```

Comment:
is 200 is the HTTP code for success?

There are a bunch of 20X codes that could be success as well. Are we using any of them in our APIs?

Reply:
- Yes, 200 stands for success.
- The endpoints used in LanaR all only use 200 from the 2XX range:
```
/api/v2/aggregate-data
/api/v2/mining/discover-model
(/api/discoveredModelWithFilter)
/api/logs
/api/users/by-token
/api/logs/csv
```

This might be standard for all our API endpoints, not sure though.